packages/components/form/src/form-control-mixin.ts

```diff
@@ -108,9 +114,7 @@ export function FormControlMixin<T extends Constructor<ReactiveElement>>(constru
     /** The name of the form control. */
     @property({ reflect: true }) name?: string;
 
-    /** Whether to show the validity state.
-     * @type {'valid' | 'invalid' | undefined}
```

Comment:
The type is still not properly shown, when it comes from external file, so maybe we should keep it for now? Diana was aware about that issue with cem plugin. 
<img width="913" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/366e80ed-3f6e-44e3-897f-73586982d3fd">

Reply:
See other comment: probably related to build issue with website.